vaadin-notification-flow/src/main/java/com/vaadin/flow/component/notification/GeneratedVaadinNotification.java

```diff
@@ -31,7 +31,40 @@
  * </p>
  * <p>
  * {@code <vaadin-notification>} is a Web Component providing accessible and
- * customizable notifications (toasts).
+ * customizable notifications (toasts). The content of the notification can be
```

Comment:
Nit: not sure if all of this info should get to the JavaDocs as it's mostly about the webcomponent, but not a blocker as it's auto-generated.

Reply:
The convention is that the generated files should be exactly what the generator outputs. Otherwise we would need to re-apply the manual changes each time after updating the generated class.